### DISCUSSION TITLE
Sicherheitslücke in der Hoymiles Cloud

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#1388] [Q&A]
Schwere Sicherheitslücke in der Hoymiles Cloud:

https://www.heise.de/news/Hoymiles-Bedrohliche-Luecken-in-der-S-Miles-Cloud-9319500.html

**Comment by softwm8818** [comment]
betrifft doch nur orignal DTU und nicht openDTU also 🤷‍♂️

**Reply by DPO99** [reply]
Ist doch gute Werbung und zeigt mal wieder, wieso so viele nicht in die China Cloud wollen...

**Comment by Fokker-1** [comment]
> betrifft doch nur orignal DTU und nicht openDTU also 🤷‍♂️

... so hatte ich es ja auch geschrieben:
Sicherheitslücke in der **Hoymiles Cloud**

**Comment by Fokker-1** [comment]
> Ist doch gute Werbung und zeigt mal wieder, wieso so viele nicht in die China Cloud wollen...

... das sehe ich auch so.
